lib/linguist/languages.yml

```diff
@@ -1411,9 +1411,14 @@ Groff:
   - .n
   - .rno
   - .roff
-  tm_scope: text.groff
+  - .tmac
+  filenames:
+  - mmn
```

Comment:
Hah! Nope, definitely not. ;)

They all just seem to be dates for some reason. :|